<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BU_Abylsen</class>
 <widget class="QMainWindow" name="BU_Abylsen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1171</width>
      <height>761</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1171</width>
      <height>761</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1171</width>
      <height>761</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="adm">
     <attribute name="title">
      <string>adm</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>IA</string>
        </property>
        <widget class="QTableView" name="IAsList">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>531</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="addNewIA">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>370</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Add New</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>410</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Ing</string>
        </property>
        <widget class="QTableView" name="INGsList">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>521</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="addNewING">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>370</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Add New</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>420</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>activite</string>
     </attribute>
     <widget class="QCalendarWidget" name="activityCalendar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>1101</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableView" name="activityList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>1071</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>681</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QListView" name="Activity_IAsList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>261</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Business</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget_4">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>120</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
